guides/common/modules/con_host-management-and-monitoring-using-cockpit.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Host_Management_and_Monitoring_Using_Cockpit_{context}"]
-= Host Management and Monitoring Using {CockpitTitle}
+= Host management and monitoring using {CockpitTitle}
```

Comment:
I am wondering if we should keep "SmartProxyServerTitle" and "CockpitTitle"? Maybe those attributes are no longer is use after this PR.

Reply:
Good point. The attributes should be reviewed from time to time.